### PR TITLE
[ Bug ] Free content of g_file_get_contents

### PR DESCRIPTION
**Pull request by jijoongmoon**
Add free when return

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: nnstreamer/tensor_filter/tensor_filter_nntrainer.cc [hunk 1]
```diff
@@ -176,6 +176,7 @@ int NNTrainer::loadModel() {
 
   if (!g_file_get_contents(model_config, &content, &file_size, &file_error)) {
     ml_loge("Error reading model config!! - %s", file_error->message);
+    g_free(content);
```
Comment (kparichay) [inline nnstreamer/tensor_filter/tensor_filter_nntrainer.cc:179]: this is not needed.
As the [documentation](https://developer.gnome.org/glib/stable/glib-File-Utilities.html#g-file-get-contents) says, if error, content will be set to NULL.

**Review by kparichay — APPROVED** [review]
LGTM

**Review by zhoonit — APPROVED** [review]
LGTM

File: nnstreamer/tensor_filter/tensor_filter_nntrainer.cc (hunk 1 above)
Comment (jijoongmoon) [reply nnstreamer/tensor_filter/tensor_filter_nntrainer.cc:179]: > this is not needed but will work with it too as `g_free()` accepts NULL.
> As the [documentation](https://developer.gnome.org/glib/stable/glib-File-Utilities.html#g-file-get-contents) says, if error, content will be set to NULL.

Good. then, I will remove this g_free.

**Pull request merged**